jira/resources.py

```diff
@@ -482,6 +482,13 @@ def permalink(self):
     def __eq__(self, other):
         return self.id == other.id
 
+    def __getitem__(self, item):
+        # Grab the JIRA server
+        jira_server = self._session.jira_server
+
+        # Look for the appropriate fields list on the server
+        return getattr(self.fields, jira_server._fields[item])
```

Comment:
One thought: Would it be worth expanding out this list on construction and populating all the custom fields as attributes of `self.fields` on construction?

This way you don't have to know if a given field is a custom field or not -- they will be treated equally.

Also means that you would able to see in the debugger what custom fields are available in an `Issue`, which is a little bit nicer for interactive development.

(Also would mean not having to reach into internals of the `JIRA` instance potentially a long time after `Issue` construction.)

Could use an intersection of `_fields.keys()` with `dir(self.fields)` to get the list of attributes to iterate through and set.